Library/src/Systems/Assets/AssetSystem.cpp

```diff
@@ -142,6 +150,22 @@ std::shared_ptr<chs::PrototypeDto> CreatePrototypeDto(const Optional<String>& Sp
 	return PrototypeInfo;
 }
 
+
+csp::common::String CreateUniqueMaterialAssetCollectionName(const csp::common::String& Name, const csp::common::String& SpaceId)
+{
+	return MATERIAL_ASSET_COLLECTION_NAME_PREFIX + SpaceId + "_" + Name;
```

Comment:
Are we confident that these names will actually be unique? For example if I passed in an asset name of car_texture. It feels like in a car themed space that might be quite a common choice. Do we do anything to guard against name collisions our end? I will probably find out the answer to that last question further down.